Form-Controls/index.html

```diff
@@ -1,28 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>T-shirt Order Form</title>
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
+  <div class="form-container">
+    <h2>T-shirt Order Form</h2>
+    <form name="tshirtForm">
+      <label for="name">Customer's Name:</label>
+      <input type="text" id="name" name="name" required minlength="2" pattern="[A-Za-z]+">
+      <div class="error-message">Please enter a valid name </div>
+
+      <label for="email">Customer's Email:</label>
+      <input type="email" id="email" name="email" required>
+      <div class="error-message">Please enter a valid email address.</div>
+
+      <label for="color">T-shirt Color:</label>
+      <select id="color" name="color" required>
+        <option value="" disabled selected>Select a color</option>
+        <option value="red">Red</option>
+        <option value="blue">Blue</option>
+        <option value="green">Green</option>
+      </select>
+      <div class="error-message">Please select a T-shirt color.</div>
+
+      <label for="size">T-shirt Size:</label>
+      <select id="size" name="size" required>
+        <option value="" disabled selected>Select a size</option>
+        <option value="XS">XS</option>
+        <option value="S">S</option>
+        <option value="M">M</option>
+        <option value="L">L</option>
+        <option value="XL">XL</option>
+        <option value="XXL">XXL</option>
+      </select>
+      <div class="error-message">Please select a T-shirt size.</div>
+
+      <label for="orderingTime">Ordering Time:</label>
+      <input type="text" id="orderingTime" name="orderingTime" readonly>
+
+      <label for="deliveryTime">Estimated Delivery Time:</label>
+      <input type="text" id="deliveryTime" name="deliveryTime" readonly>
+
+      <input type="submit" value="Submit">
+    </form>
+  </div>
+
+  <script>
+    document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Does this meet the criteria listed in the readme? Can you explain how it does, or what your goal in with including this?